bitnami/fluentd/templates/forwarder-daemonset.yaml

```
@@ -44,6 +44,10 @@ spec:
       {{- if .Values.forwarder.securityContext.enabled }}
       securityContext: {{- omit .Values.forwarder.securityContext "enabled" | toYaml | nindent 8 }}
       {{- end }}
+      {{- if .Values.forwarder.extraInitContainers }}
+      initContainers:
+      {{- include "fluentd.tplValue" ( dict "value" .Values.forwarder.extraInitContainers "context" $ ) | nindent 8 }}
```

```suggestion
      initContainers: {{- include "fluentd.tplValue" ( dict "value" .Values.forwarder.extraInitContainers "context" $ ) | nindent 8 }}
```

As we are using `nindent`, we can leave this in only one line.